client/src/assets/components/profile/EditProfile.jsx

```diff
@@ -136,6 +162,13 @@ const EditProfile = () => {
           headers: { Authorization: `Bearer ${auth?.accessToken}` },
           withCredentials: true
         })
+      if (response?.status === 200) {
+        showSuccess(`${field} updated successfully`);
+        setErrMsg('');
+      }
```

Comment:
This is a great addition but {field} might not be too user friendly in the future when we hit production.

Consider using:
```
showSuccess(`Profile updated successfully.`);
```
For the moment while there's a work around for "First Name" updated successfully instead of firstName for instance.

Also since you'll be adding the error toast as well, some logic can be removed through the entire file relating to errMsg, setErrMsg